src/main/java/com/bandwidth/sdk/messaging/models/MessageEvent.java

```diff
@@ -18,6 +20,17 @@
     public abstract Optional<Integer> getErrorCode();
     public abstract Optional<String> getTo();
 
+
+    @JsonIgnore
+    public boolean isIncomingMessage(){
+        return getTo().isPresent();
```

Comment:
All messages have "to" numbers, this doesn't determine if it's incoming. You should look at the event type instead

Reply:
I think the logic here is that the callback won't have a "to" field if the callback isn't an incoming message. I'm not sure if that is the case 100% of the time. All callbacks have a "message" field which has a "message.to" field, which is different than the callback's "to" field.

Not saying I disagree with looking at the event type though

Reply:
@jmulford-bandwidth you are correct, I thought this was on the message object itself, but this is the message event. It's still probably better to look at the event type, there may be cases where the "to" field could exist for other types (potentially in the future). We would just have a constant "message-received" and check against that